app/routes.js

```diff
@@ -92,6 +92,18 @@ export const routes = (authProvider) => {
     res.render('pages/sections/supplier/search/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.post('/organisation/:orderId/supplier/search', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
+    const { orderId } = req.params;
+
+    const response = validateSupplierSearchForm({ data: req.body });
+
+    const context = await getSupplierSearchPageErrorContext({
+      orderId,
+      validationErrors: response.errors,
+    });
+    return res.render('pages/sections/supplier/search/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
```

Comment:
are you doing the happy path in a separate ticket? ie `success: true`

Reply:
👍 yes